pkg/logs/input/container/docker.go

```diff
@@ -211,7 +212,7 @@ func (dt *DockerTailer) keepDockerTagsUpdated() {
 }
 
 func (dt *DockerTailer) checkForNewDockerTags() {
-	tags, err := tagger.Tag(dockerutil.ContainerIDToEntityName(dt.containerId), true)
```

Comment:
Why duplicate code instead of importing the dockerutil package? Our goal for the infra agent is to remove all usages of `docker/docker/client` in leaf packages and move all the docker logic in `dockerutils`, to handle all docker idiosyncrasies (empty objects returned on inspects, image sha instead of name...) only once.

If this static function is needed in scenarios without the `docker` build tag, we can move it to `docker_common.go` which has no build tag

Reply:
okay, fixed